ui/components/component-library/avatar-badge/avatar-badge.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box/box';
+
+export const badgePositions = ['top', 'bottom'];
+
+export const AvatarWithBadge = ({
+  children,
+  badgePosition,
+  badge,
+  ...props
+}) => {
+  return (
+    <Box className="avatar-badge" {...props}>
+      {/* Jazzicon , Avatar Account */}
+      {children}
+
```

Comment:
Possibly want to update the docs here. Not sure if we need a line break after {children} unless it's a lint thing :)
```suggestion
      {/* Generally the AvatarAccount */}
      {children}
```